src/Attribute.cpp

```diff
@@ -0,0 +1,65 @@
+#if HAVE_CONFIG_H
+#  include <config.h>
+#endif
+
+#ifdef WIN32
+#pragma warning(disable : 4786)   /* Disable annoying warning messages */
+#endif
+
+#include "random.h"
+#include "OutputMgr.h"
+#include "Attribute.h"
+
+void
+AttributeGenerator::GenerateAllAttributes(std::ostream &out)
```

Comment:
The attributes are for functions only as of now? Maybe rename this to "FunctionAttribute::Output"

Reply:
Yes. But keeping it as `AttributeGenerator` seems appropriate as I want to send type, variable and label attributes also.

Changed `GenerateAllAttributes` to `Output`.